packages/orchestration/src/exos/chain-hub-admin.js

```diff
@@ -46,8 +46,8 @@ export const prepareChainHubAdmin = (zone, chainHub) => {
        * Register information for a chain
        *
        * @param {string} chainName - must not exist in chainHub
-       * @param {CosmosChainInfo} chainInfo
-       * @param {IBCConnectionInfo} connectionInfo - from Agoric chain
+       * @param {ChainInfo & { chainId: string }} chainInfo
+       * @param {IBCConnectionInfo} [connectionInfo] - from Agoric chain
```

Comment:
consider `ibcConnectionInfo` to make more clear it applies only for that case.

we'll probably want to refactor this more eventually so the canonical method does't assume any ecosystem/namespace